db/migration/1739195264083-AddDataInsightFolderIds.ts

```diff
@@ -0,0 +1,103 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+const dataInsightFolders = [
+    {
+        fullName: "Veronika Samborska",
```

Comment:
I've mentioned this new dependency to Ed so that he can add it as a TODO to his onboarding checklist 🙂

Reply:
Thanks!